reactioncommerce/reaction
# Make all Meteor.Error calls consistent

Resolves #3122 

## How To Test
In an editor, search for all instances of `Meteor.Error` check that it conforms with this style `Meteor.Error("short-message", "Long message with more detail")`
